platform/src/main/java/org/stellar/anchor/platform/component/share/ClientsBeans.java

```diff
@@ -3,13 +3,29 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.stellar.anchor.api.exception.InvalidConfigException;
+import org.stellar.anchor.client.ClientService;
+import org.stellar.anchor.client.DefaultClientService;
+import org.stellar.anchor.config.ClientsConfig;
 import org.stellar.anchor.platform.config.PropertyClientsConfig;
+import org.stellar.anchor.platform.config.PropertyClientsConfig_DEPRECATED;
 
 @Configuration
 public class ClientsBeans {
   @Bean
   @ConfigurationProperties(prefix = "")
-  PropertyClientsConfig clientsConfig() {
+  PropertyClientsConfig_DEPRECATED clientsConfigDeprecated() {
+    return new PropertyClientsConfig_DEPRECATED();
+  }
+
+  @Bean
+  @ConfigurationProperties(prefix = "clients")
```

Comment:
What happens if the new client config is not provided? Does Anchor Platform still start?

Reply:
It should since ClientsConfig is not a required config.
If it's not provided we will just assume it's an empty config